crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -1317,6 +1317,55 @@ module deepbook::clob_v2 {
         }
     }
 
+    /// Clean up expired orders
+    /// Note that this function can reduce gas cost if  orders
+    /// with the same price are grouped together in the vector.
+    /// For example, if we have the following order_id to price mapping, {0: 100., 1: 200., 2: 100., 3: 200.}.
+    /// Grouping order_ids like [0, 2, 1, 3] would make it the most gas efficient.
+    /// Order owners should be the owner addresses from the account capacities which placed the orders,
+    /// and they should correspond to the order IDs one by one.
+    public fun clean_up_expired_orders<BaseAsset, QuoteAsset>(
+        pool: &mut Pool<BaseAsset, QuoteAsset>,
+        clock: &Clock,
+        order_ids: vector<u64>,
+        order_owners: vector<address>
```

Comment:
Do we want to allow Alice to clean up an order from Bob?

Reply:
Anyone can clean up the order from bob, so does not matter whether it is the owner or alice or someone else.